diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -202,6 +203,11 @@ namespace diff_drive_controller
     use_position_ =  pose_from_joint_position_ ||  twist_from_joint_position_;
     use_velocity_ = !pose_from_joint_position_ || !twist_from_joint_position_;
 
+    controller_nh.param("period_from_time", period_from_time_, period_from_time_);
+    ROS_INFO_STREAM_NAMED(name_,
+        "Control period will be computed using the " <<
+        (period_from_time_ ? "time" : "period") << ".");
```

Comment:
Can this be made more explicit? Eg,

```
ROS_INFO_STREAM_NAMED(name_,
        "Control period will be" <<
        (period_from_time_ ? "computed from delta in update() Time values" : "duration passed to update()") << ".");
```

I feel like this would clarify a bit better what is actually being configured.